cmd/main.go

```diff
@@ -60,12 +61,13 @@ required arguments:
 			Port:              *port,
 			Debug:             *debug,
 		},
-		qbit,
-		http.DefaultClient)
+		qbit)
 
 	if err != nil {
 		panic(err)
```

Comment:
`panic` should only be used in the most extreme situations. I would recommend a `log.Fatal` here instead. It will still exit the program, just a little bit more gracefully

Reply:
Done :)